environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python>=3.6,<3.8
+  - python>=3.6
```

Comment:
Does 3.6 work? Since the expertsystem requires > 3.6

Reply:
No, but that will be tackled by #113. In this PR, we still use `expertsystem==0.2.0`